app/services/album.js

```diff
@@ -0,0 +1,20 @@
+const axios = require('axios'),
+  config = require('../../config'),
+  base_uri = config.common.albumsApi.url,
+  logger = require('../logger');
+
+exports.getAlbum = params =>
+  axios
+    .get(`${base_uri}/albums?id=${params.id}`)
```

Comment:
you can use: `${base_uri}/albums/${params.id}` in order to receive not an array but a single element

Reply:
this is great i didn't know